src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard/MovieCard';
+
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[],
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```

Comment:
it's required prop, no need for default param
```suggestion
export const MoviesList: React.FC<Props> = ({ movies }) => (
```